packages/paste-core/components/combobox/__tests__/virtualization.spec.tsx

```diff
@@ -39,17 +39,17 @@ describe('Combobox - Virtualization', () => {
   describe('Without option template', () => {
     const items = ['a', 'b', 'c', 'd', 'e'];
     it('should apply the correct height to the virtualized list box', () => {
-      render(<Combobox data-testid="test" items={items} labelText="Choose a letter:" />, {
+      render(<Combobox data-testid="test" items={items} labelText="Choose a letter:" initialIsOpen />, {
```

Comment:
fyi: I want to say I ran into this previously, but I'm not 100% sure. when debugging those jest memory leaks, it looked like setting initial open was causing an state change that wasn't getting wrapped in `act`. I could also be totally thinking of something else.

Reply:
I fixed the leak